core/src/main/java/org/apache/accumulo/core/spi/scan/ConfigurableScanServerSelector.java

```diff
@@ -83,13 +83,25 @@
  * certain scans. Second groups can be used to have different hardware/VM types for scans, for
  * example could have some scans use expensive high memory VMs and others use cheaper burstable
  * VMs.</li>
- * <li><b>enableTabletServerFallback : </b> When there are no scans servers, this setting determines
- * if fallback to tablet servers is desired. Falling back to tablet servers may cause tablets to be
- * loaded that are not currently loaded. When this setting is false and there are no scan servers,
- * it will wait for scan servers to be available. This setting avoids loading tablets on tablet
- * servers when scans servers are temporarily unavailable which could be caused by normal cluster
- * activity. If not specified this setting defaults to true. Set to false to avoid tablet server
- * fallback. Waiting for scan servers is done via
+ * <li><b>timeToWaitForScanServers : </b> When there are no scans servers, this setting determines
+ * how long to wait for scan servers to load before falling back to tablet servers is desired.
+ * Falling back to tablet servers may cause tablets to be loaded that are not currently loaded. When
+ * this setting is given a wait time and there are no scan servers, it will wait for scan servers to
+ * be available. This setting avoids loading tablets on tablet servers when scans servers are
+ * temporarily unavailable which could be caused by normal cluster activity. You can specify the
+ * wait time using different units to precisely control the wait duration. The supported units are:
+ * <ul>
+ * <li>"d" for days</li>
+ * <li>"h" for hours</li>
+ * <li>"m" for minutes</li>
+ * <li>"s" for seconds</li>
+ * <li>"ms" for milliseconds</li>
+ * </ul>
+ * If duration is not specified this setting defaults to 0s, and will disable the wait for scan
+ * servers. When set to a large value, the system will effectively wait for scan servers to become
```

Comment:
The word "system" is ambiguous. I would recommend using either "client" or "selector" to denote that this logic is handled by the client attempting the scan.